gazebo_plugins/include/gazebo_plugins/gazebo_ros_force.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright 2012 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAZEBO_PLUGINS__GAZEBO_ROS_FORCE_HPP_
+#define GAZEBO_PLUGINS__GAZEBO_ROS_FORCE_HPP_
+
+#include <gazebo/common/Plugin.hh>
+#include <geometry_msgs/msg/wrench.hpp>
+
+#include <memory>
+
+namespace gazebo_plugins
+{
+class GazeboRosForcePrivate;
+
+/// This plugin collects data from a ROS topic and applies wrench to a link accordingly.
+/**
+  \details The last received force will be continuously added to the link at every simulation iteration.
+  Send an empty / zero message to stop applying a force.
+
+  Example Usage:
+  \verbatim
```

Comment:
I think ```\code{.xml}``` shows up better

Reply:
It sounds like it should, so changed https://github.com/ros-simulation/gazebo_ros_pkgs/pull/784/commits/0cc0e06f4aaca8b9cabd025dde907402afc9fc7a